usaspending_api/conftest.py

```diff
@@ -123,6 +123,15 @@ def django_db_setup(
             ensure_business_categories_functions_exist()
             call_command("load_broker_static_data")
 
+            # temporarily updating the environment so commands in tests will point to the test database
+            old_environ = os.environ.copy()
+            os.environ["DATABASE_URL"] = generate_test_db_connection_string()
+
+    def reset_environ():
```

Comment:
Maybe add clarifying comments that this is called by the finalizer below, and use of a finalizer is basically like auto-baking in a "tear-down" behavior to every test (if that is true).